grep-00XX-relicense-volk.md

```diff
@@ -0,0 +1,86 @@
+# GREP [00XX] -- [Relicense VOLK]
+
+- Original Author: [Johannes Demel <demel@uni-bremen.de>]
+- Champion: [Johannes Demel <demel@uni-bremen.de>]
+- Status: Draft
+
+History:
+- 18-Mar-2021: Initial Draft
+
+## Abstract
+
+We have several community requests to relicense VOLK under a more permissive license.
+The proposed new license is [LGPLv3+](https://www.gnu.org/licenses/lgpl-3.0.html).
+Historically, VOLK was part of GNU Radio. Thus, it made total sense to use GPLv3+ just like GNU Radio.
+Since then, VOLK became its own project with its own repository and leadership.
+While it is still a core dependency of GNU Radio and considers GNU Radio as its main user, others may want to use it too.
+Especially, long term VOLK contributors expressed their desire to be able to use VOLK in a broader set of projects.
+
+### State of the GREP
+This GREP does currently serve as a platform to collect all the necessary information and ideas to relicense VOLK.
+Everyone is invited to contribute to this GREP with ideas, pull requests, comments etc.
+
+## Copyright / License
+
+This GREP is licensed as CC-BY-ND.
+Copyright 2021 Johannes Demel and any co-author listed above.
+
+
+## Motivation
+
+We recognize the desire of our long term contributors to be able to use VOLK with their contributions in their projects.
+This may explicitly include proprietary projects.
+We want to enable all contributors to be able to use VOLK wherever they want.
+At the same time, we want to make sure that improvements to VOLK itself are easily obtained by everyone, i.e. strike a balance between permissiveness and strict copyleft.
+Previously, several attemps were made to relicense VOLK without success.
+With a formal process and a clear vision laid out, we hope to be able to get everyone on board.
+
+Since VOLK is a library it should choose a fitting license.
+If we see greater adoption of VOLK in more project, we hope to also see more contributions.
+May it be bug fixes, new kernels or even contributions to core features.
+
+## Description
+
+In order to facilitate this relicensing effort, we need to got through several steps.
+
+### Notification
+We consider it to be very important to let every contributor know that the wish to relicense by some contributors exists.
+A first step would be to notify every contributor of this GREP.
+
+### Another license discussion
```

Comment:
Do we really want to foster that discussion here? I assume you've got your own ideas, and you've talked to those who can't contribute under GPLv3, so you've already made up your mind what the best license is. If we explicitly invite that discussion here, we'll have years of fun discussing things. The discussion is open, anyways: If people don't agree to LGPL3+, they can comment on this grep.

Reply:
So you suggest to rewrite this GREP such that it says: We want to relicense to LGPLv3+, these are our reasons. My wording suggests that this is an open discussion.

Reply:
Thanks for your insights here. I updated this section accordingly.